examples/cli-instrumentation/instrumentation.ts

```diff
@@ -1,8 +1,9 @@
 import { createLogger } from "apibara/runtime";
 import type { LoggerFactoryFn, RegisterFn } from "apibara/types";
+import { sdk } from "./otel/otel";
```

Comment:
Same comment as the documentation PR. Let's keep everything in one file if its content is not reused.